sodalite-server/src/main/java/org/icgc/dcc/sodalite/server/model/analysis/variantCall/VariantCall.java

```diff
@@ -1,15 +1,17 @@
 
-package org.icgc.dcc.sodalite.server.model;
+package org.icgc.dcc.sodalite.server.model.analysis.variantCall;
 
 import java.util.HashMap;
 import java.util.Map;
+
+import org.apache.commons.lang.builder.ToStringBuilder;
```

Comment:
Not critical at all :-) ... it's just how all of our other toString methods for entities are currently constructed, so I just kept with the pattern of the existing code. We can refactor a LOT of ugly entity code if we want to. :-)